dataflow/tests/live-variable/Expected.txt

```diff
@@ -66,23 +71,75 @@ b   [ LocalVariable ]
 (a + b)   [ NumericalAddition ]
 e = (a + b)   [ Assignment ]
 ~~~~~~~~~
-TransferInput#5
+TransferInput#32
 After:   live variables = a
 
 11:
 Process order: 6
 AnalysisResult#11
 Before:   live variables = a
 ~~~~~~~~~
+f   [ VariableDeclaration ]
+b   [ LocalVariable ]
+0   [ IntegerLiteral ]
+b = 0   [ Assignment ]
+marker (start of try statement #0)   [ Marker ]
+marker (start of try block #0)   [ Marker ]
+f   [ LocalVariable ]
+1   [ IntegerLiteral ]
+a   [ LocalVariable ]
+~~~~~~~~~
+TransferInput#21
+After:   live variables = a, b
```

Comment:
Before the logic change in `BackwardAnalysisImpl`, the live variable set here would incorrectly be missing `b`